asserts/account_key.go

```diff
@@ -110,6 +110,13 @@ func (ak *AccountKey) checkConsistency(db RODatabase, acck *AccountKey) error {
 // sanity
 var _ consistencyChecker = (*AccountKey)(nil)
 
+// Prerequisites returns references to this account key prerequisite assertions.
```

Comment:
for consistency, maybe "… account-key's prerequisite …" on this doc comment and add the apostrophe on the other, similar comments?